### PR TITLE
Add mask step

### PR DESCRIPTION
**Pull request by aquatiko**
Fixes https://github.com/JuliaImages/ImageSegmentation.jl/issues/22

### CODE REVIEW

File: docs/src/imagesegmentation.md [hunk 1]
```diff
@@ -451,6 +451,8 @@ julia> segments = watershed(dist, markers)
 Segmented Image with:
   labels map: 312×252 Array{Int64,2}
   number of labels: 24
+
+julia> segments = imshow( map(i->segment_mean(segments,i), labels_map(segments)) .* .~bw)       #shows segmented image
```
Comment (Evizero) [inline docs/src/imagesegmentation.md]: I don't know this code snippet, but the `segments = ...` prefix doesn't quite look right

Comment (aquatiko) [reply docs/src/imagesegmentation.md]: @Evizero Updated. The code snippet is from the docs only, I have just added a mask to show the segmented image.

**Comment by tejus-gupta** [comment]
@aquatiko Thanks for the PR.

The image in the docs was obtained by replacing each label with a random colour. Better to update the example code to this.

```
using ImageView, Random
function get_random_color(seed)
    Random.seed!(seed)
    rand(RGB{N0f8})
end

# remove labels from background region
imshow(map(i->get_random_color(i), labels_map(segments).*(1 .-bw)))
```

**Comment by tejus-gupta** [comment]
LGTM!

**Pull request merged**